core/deployment/src/main/java/io/quarkus/deployment/util/WebJarUtil.java

```diff
@@ -104,15 +106,15 @@ public static Path copyResourcesForDevOrTest(LiveReloadBuildItem liveReloadBuild
             String rootFolderInJar)
             throws IOException {
         return copyResourcesForDevOrTest(liveReloadBuildItem, curateOutcomeBuildItem, launchMode, resourcesArtifact,
-                rootFolderInJar, true);
+                rootFolderInJar, true, false);
     }
 
     public static Path copyResourcesForDevOrTest(LiveReloadBuildItem liveReloadBuildItem,
```

Comment:
I would appreciate if you could add a deprecated version of the old method calling this one as I'm using this in one of my extensions.

Reply:
Sure, was maybe to optimistic to directly change method signatures. Will add a deprecated one.

Reply:
done